compal/__init__.py

```diff
@@ -609,33 +804,60 @@ class Diagnostics(object):
     def __init__(self, modem):
         self.modem = modem
 
-    def test_ping(self, target_addr, ping_size=64, num_ping=3, interval=10):
+    def start_pingtest(self, target_addr, ping_size=64, num_ping=3,
+                       interval=10):
         """
-        Ping
+        Start Ping-Test
         """
-        res = self.modem.xml_setter(Set.PING_TEST, OrderedDict([
+        return self.modem.xml_setter(Set.PING_TEST, OrderedDict([
             ('Type', 1),
```

Comment:
On my modem (IPv4 only) I needed `Type=0` for ping to work.

See [my version](https://github.com/ties/compal_CH7465LG_py/blob/feature/diagnostics_wip/compal/__init__.py#L617) of the diagnostics that I forgot to push last weekend